### ISSUE TITLE
Point with a description - "Point has no descriptive tags"

### ISSUE THREAD

**Issue by jsavage** [openstreetmap/iD#9178]
### How to reproduce the issue?

Add a point
Add Feature type: point
Add Tag Description: White spherical buoy used as a racing not for navigation
Add Tag Name: ASC
under Issues it says Point has no descriptive tags

### Screenshot(s) or anything else?

![image](https://user-images.githubusercontent.com/189524/175246806-3dfcf96b-efcf-4ced-8ecc-6e7d1569a04e.png)

### Which iD Editor versions do you see the issue on?

Released version at openstreetmap.org/edit

### Which browsers are you seeing this problem on?

Chrome

**Comment by danieldegroot2** [comment]
This is working as intended. Objects without a feature type can not be easily used by a data user or [verified](https://wiki.openstreetmap.org/wiki/Verifiability) by other contributors, or [may not belong](https://wiki.openstreetmap.org/wiki/Good_practice) in the OSM database -at least not directly-. You can use the [notes](https://wiki.openstreetmap.org/wiki/Notes) feature if you're unsure of how to map something.

You can use [Any tags you like](https://wiki.openstreetmap.org/wiki/Any_tags_you_like) (ATYL).
tldr; Any [primary key](https://wiki.openstreetmap.org/wiki/Map_features) (`building`, `highway`, `landuse`, etc.) describes the feature type. You can use any value to specify the feature type further, ie. somewhat bad example `seamark:type=racing_buoy` -this will make it an actual tag and get rid of the warning-. Usually a tag already exists, though.
(see also [Tags#Finding your tag](https://wiki.openstreetmap.org/wiki/Tags#Finding_your_tag))

You seem to have come across a special purpose buoy. It [has been mapped](https://www.openstreetmap.org/node/1724331226) with [`seamark:buoy_special_purpose:category=warning`](https://wiki.openstreetmap.org/wiki/Tag:seamark:buoy_special_purpose:category%3Dwarning).
(Sidenote: there's also an old, unused tag [`seamark:buoy_special_purpose:category=racing`](https://wiki.openstreetmap.org/wiki/Tag:seamark:buoy_special_purpose:category%3Dracing).)

(btw, seamark tagging can look different from tagging other objects, as these tags come from a different project, OpenSeaMap, but are added through OpenStreetMap.)

If this resolves the issue, feel free to close it yourself. You can find the purple Close issue button below alongside the green Comment button.

**Comment by jsavage** [comment]
Thanks, lets see if I have understood.  Is the following likely to be acceptable?  If so, I think I have a way forward.
![image](https://user-images.githubusercontent.com/189524/175436214-c7e27daf-4ea9-44fb-b12f-f83417769940.png)

**Comment by danieldegroot2** [comment]
What is "ASC"? In the top post you mentioned it as name, here as source. If it is the racing club/group operating it, you can tag them as [`operator`](https://wiki.openstreetmap.org/wiki/Key:operator). Try to [avoid abbreviations](https://wiki.openstreetmap.org/wiki/Names#Abbreviation_(don't_do_it)).

If the buoy does not have an actual name, only a number or some number-letter combination, you should [tag as `ref` instead](https://wiki.openstreetmap.org/wiki/Key:ref). This is a common mistake, which was also present in the example provided.
(from the same page, sections [Name is the name only](https://wiki.openstreetmap.org/wiki/Names#Name_is_the_name_only) and [Names are not for descriptions](https://wiki.openstreetmap.org/wiki/Names#Names_are_not_for_descriptions).)
